postman/config.yml

```diff
@@ -0,0 +1,39 @@
+# Name of the quickstart (required)
+name: Postman
+
+# Displayed in the UI (required)
+title: Postman
+
+# Long-form description of the quickstart (required)
+description: |
+  Observe the performance of your APIs using Postman and New Relic
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Observe the performance of your APIs using Postman and New Relic
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Shashank Awasthi
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - API
+  - Postman
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - infra-agent-standard
```

Comment:
This won't work for installing the quickstart, you will have to create an install file. You can add a new folder in the install/third-party directory. You can copy another install file there as a template.

Reply:
Added the directory and updated the config file